test/functional/Java11andUp/src/org/openj9/test/condy/GarbageCollectionCondyTest.java

```diff
@@ -45,13 +45,18 @@ public void testNullCondyUnloading() {
 		return;
 	}
 
-
 	@Test(groups = { "level.sanity" })
 	public void testExceptionCondyUnloading() {
 		byte[] classBytes = DefineCondyClass.getExceptionCondyClassBytes("test/exceptionClass");
 		runTest(classBytes, "exceptionClass");
 		return;
 	}
+
+	@Test(groups = { "level.sanity" })
+	public void testCondyStackMap() {
+		PrimitiveCondyMethods.stackmapTest();
```

Comment:
A nitpick name change to prevent confusion with classfile StackMapTables

```suggestion
		PrimitiveCondyMethods.stackmapperTest();
```